auditbeat/buildkite.yml

```diff
@@ -0,0 +1,50 @@
+when:
+    changeset:                 ## when PR contains any of those entries in the changeset
+        - "^auditbeat/.*"
+        - "@ci"                ## special token regarding the changeset for the ci
+        - "@oss"               ## special token regarding the changeset for the oss
+stages:
+    # mandatory stage - it runs always for:
+    #   - branches/tags
+    #   - on PRs
+    #   - GitHub comment /test auditbeat
+    #   - GitHub label auditbeat
+    mandatory:
+        # NOTE: stage name should be unique!
+        unitTest:
+            command: "mage build unitTest"
+            platform: "family/core-ubuntu-2204"
+        crosscompile:
+            command: "make -C auditbeat crosscompile"
+            platform: "family/core-ubuntu-2204"
+        rhel-9:
+            command: "mage build unitTest"
+            platform: "family/core-rhel-9"
+        unitTest-windows-2022:
+            command: "mage build unitTest"
+            platform: "windows-2022"
+        unitTest-windows-2016:
+            command: "mage build unitTest"
+            platform: "family/core-windows-2016"
+    # optional stage - it runs on:
+    #   - branches/tags
+    #   - on PRs if:
+    #       - GitHub comment /test auditbeat <sub-stage-name> . i.e: /test auditbeat integTest
+    #       - GitHub label <sub-stage-name> . i.e: integTest or unitTest-arm or unitTest-macos ...
+    extended:
+        # NOTE: stage name should be unique!
+        integTest:
+            command: "mage build integTest"
+            platform: "core-ubuntu-2004-aarch64"
+        integTest-arm:
```

Comment:
This step can be forced to run using:

* The GitHub comment `/test auditbeat integTest-arm`
* The GitHub label `auditbeat-integTest-arm`